build.sbt

```diff
@@ -20,15 +20,15 @@ val commonSettings = Defaults.coreDefaultSettings ++ Seq(
   version := "0.1.4",
   scalaVersion := "2.11.8",
   crossScalaVersions := Seq("2.11.0", "2.11.1", "2.11.2", "2.11.3", "2.11.4", "2.11.5", "2.11.6", "2.11.7", "2.11.8", "2.12.0", "2.12.1"),
```

Comment:
I'm not sure we want so many different Scala versions listed here. Why not just "2.11.8" and up? What does this help to have more Scala versions listed at this place?